packages/styles/components/_c.option-card.scss

```diff
@@ -0,0 +1,94 @@
+@use "sass:math";
+
+.mc-option-card {
+  $parent: get-parent-selector(&);
+
+  position: relative;
+  border-radius: get-border-radius("m");
+  cursor: pointer;
+
+  &__input {
+    position: absolute !important;
+    right: $mu100;
+    top: $mu100;
+    border-color: $color-input-border;
+
+    &.mc-radio__input,
+    &.mc-checkbox__input,
+    &:focus {
+      box-shadow: none;
+    }
+
+    &.mc-radio__input &.mc-checkbox__input &:checked {
+      border-color: $color-input-checked-border;
+    }
+
+    &.mc-radio__input &.mc-checkbox__input &:hover {
+      border-color: $color-input-border;
+    }
+
+    &.mc-radio__input &.mc-checkbox__input &:checked:hover {
+      border-color: $color-input-checked-border;
+    }
```

Comment:
I don't think this code works.
But to be seen. Otherwise, it should be deleted.